### PR TITLE
Feature/37831 grant access to custom role

### PR DESCRIPTION
**Pull request by egicquel**

### CODE REVIEW

File: src/includes/MailjetMenu.php [hunk 1]
```diff
@@ -37,17 +37,7 @@ class MailjetMenu
...
-            ||
-            (current_user_can('editor') && get_option('mailjet_access_editor') == 1)
-            ||
-            (current_user_can('author') && get_option('mailjet_access_author') == 1)
-            ||
-            (current_user_can('contributor') && get_option('mailjet_access_contributor') == 1)
-            ||
-            (current_user_can('subscriber') && get_option('mailjet_access_subscriber') == 1)
-        ) {
+        if (user_can(wp_get_current_user(), UserAccessSettings::ACCESS_CAP_NAME)) {
```
Comment (MoufidTaleb) [inline src/includes/MailjetMenu.php]: Prefer use current_user_can(UserAccessSettings::ACCESS_CAP_NAME) =D

File: src/includes/MailjetUpdate.php [hunk 2]
```diff
@@ -81,4 +73,17 @@ public static function updateToV5_2() {
         add_option('mailjet_woo_checkout_checkbox', $activateMailjetWooSync);
         delete_option('activate_mailjet_woo_sync');
     }
+
+    public static function updateToV5_3() {
+        $pluginVersion = get_option('mailjet_plugin_version');
+        if (empty($pluginVersion)) {
```
Comment (MoufidTaleb) [inline src/includes/MailjetUpdate.php]: !empty here =D

File: src/includes/MailjetMenu.php (hunk 1 above)
Comment (egicquel) [reply src/includes/MailjetMenu.php]: Will change in next commit

File: src/includes/MailjetUpdate.php (hunk 2 above)
Comment (egicquel) [reply src/includes/MailjetUpdate.php]: Oops

**Pull request merged**